ERROR - 2024-12-01 03:30:54 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:30:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:31:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:31:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:31:45 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:31:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:31:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:31:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:33:24 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gm...', 'user_email', Array, NULL, 'email')
#6 C:\laragon\www\ci4_dina\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gm...', Array, Array, 'email')
#7 C:\laragon\www\ci4_dina\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\laragon\www\ci4_dina\app\Controllers\Register.php(27): CodeIgniter\Controller->validate(Array)
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Register->save()
#10 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gm...', 'user_email', Array, NULL, 'email')
#5 C:\laragon\www\ci4_dina\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gm...', Array, Array, 'email')
#6 C:\laragon\www\ci4_dina\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\laragon\www\ci4_dina\app\Controllers\Register.php(27): CodeIgniter\Controller->validate(Array)
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Register->save()
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-12-01 03:33:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gmail.com', 'user_email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Register.php(27): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Register->save()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:34:13 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gm...', 'user_email', Array, NULL, 'email')
#6 C:\laragon\www\ci4_dina\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gm...', Array, Array, 'email')
#7 C:\laragon\www\ci4_dina\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\laragon\www\ci4_dina\app\Controllers\Register.php(27): CodeIgniter\Controller->validate(Array)
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Register->save()
#10 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gm...', 'user_email', Array, NULL, 'email')
#5 C:\laragon\www\ci4_dina\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gm...', Array, Array, 'email')
#6 C:\laragon\www\ci4_dina\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\laragon\www\ci4_dina\app\Controllers\Register.php(27): CodeIgniter\Controller->validate(Array)
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Register->save()
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-12-01 03:34:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(339): CodeIgniter\Validation\StrictRules\Rules->is_unique('dinaparker99@gmail.com', 'user_email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'dinaparker99@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Register.php(27): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Register->save()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:34:38 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:34:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:35:25 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:35:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:35:40 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:35:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:35:44 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:35:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
ERROR - 2024-12-01 03:36:16 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'login_db', 3306, '', 0)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\ci4_dina\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\ci4_dina\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\ci4_dina\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\ci4_dina\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\ci4_dina\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\ci4_dina\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\ci4_dina\system\CodeIgniter.php(933): App\Controllers\Login->auth()
#7 C:\laragon\www\ci4_dina\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\ci4_dina\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ci4_dina\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\ci4_dina\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\ci4_dina\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\ci4_dina\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-01 03:36:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login/auth]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_email` = :user_email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->auth()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\ci4_dina\\public\\index.php')
